src/providers/ad/ad_gpo.c

```diff
@@ -2447,9 +2447,15 @@ ad_gpo_process_gpo_done(struct tevent_req *subreq)
     ret = sdap_id_op_done(state->sdap_op, ret, &dp_error);
 
     if (ret != EOK && ret != ENOENT) {
-        DEBUG(SSSDBG_OP_FAILURE,
-              "Unable to get GPO list: [%d](%s)\n",
-              ret, sss_strerror(ret));
+        if (ret == EFAULT) {
+            DEBUG(SSSDBG_OP_FAILURE,
+                  "Unable to get GPO list: [%d](%s: %s)\n",
```

Comment:
Printing the server name at the end of the log line without further explanation it's not clear enough for me. I'd prefer to see `Unable to get GPO list from server %s: [%d](%s)\n`.

The same applies to other log lines.